HelpSource/Classes/Pclump.schelp

```diff
@@ -0,0 +1,76 @@
+TITLE:: Pclump
+summary:: Clumps the pattern into arrays of size n.
+categories:: Streams-Patterns-Events>Patterns>Repetition
+
+DESCRIPTION::
+Takes the source pattern and clumps it into arrays of size n, with the final element being the size of the remainder.
+
+E.g. If the source pattern has 5 elements and you choose a clump size of 2, the new pattern will return two arrays of size 2, and a final array of size 1.
```

Comment:
maybe use the description of SeqColl's `clump` as a basis? 

> Separates the collection into sub-collections by separating every groupSize elements.

Reply:
Yes that's probably a good idea.